crypto/objects/objects.txt

```diff
@@ -1124,33 +1124,33 @@ sigAlgs 16	: id-rsassa-pkcs1-v1_5-with-sha3-512	: RSA-SHA3-512
+sigAlgs 24      : id-slh-dsa-sha2-256s	: SLH-DSA-SHA2-256s
+sigAlgs 25      : id-slh-dsa-sha2-256f	: SLH-DSA-SHA2-256f
+sigAlgs 26      : id-slh-dsa-shake-128s	: SLH-DSA-SHAKE-128s
+sigAlgs 27      : id-slh-dsa-shake-128f	: SLH-DSA-SHAKE-128f
+sigAlgs 28      : id-slh-dsa-shake-192s	: SLH-DSA-SHAKE-192s
+sigAlgs 29      : id-slh-dsa-shake-192f	: SLH-DSA-SHAKE-192f
+sigAlgs 30      : id-slh-dsa-shake-256s	: SLH-DSA-SHAKE-256s
+sigAlgs 31      : id-slh-dsa-shake-256f	: SLH-DSA-SHAKE-256f
 sigAlgs 32      : id-hash-ml-dsa-44-with-sha512 : HASH-ML-DSA-44-WITH-SHA512
 sigAlgs 33      : id-hash-ml-dsa-65-with-sha512 : HASH-ML-DSA-65-WITH-SHA512
 sigAlgs 34      : id-hash-ml-dsa-87-with-sha512 : HASH-ML-DSA-87-WITH-SHA512
-sigAlgs 35      : id-hash-slh-dsa-sha2-128s-with-sha256    : SLH-DSA-SHA2-128S-WITH-SHA256
-sigAlgs 36      : id-hash-slh-dsa-sha2-128f-with-sha256    : SLH-DSA-SHA2-128F-WITH-SHA256
-sigAlgs 37      : id-hash-slh-dsa-sha2-192s-with-sha512    : SLH-DSA-SHA2-192S-WITH-SHA512
-sigAlgs 38      : id-hash-slh-dsa-sha2-192f-with-sha512    : SLH-DSA-SHA2-192F-WITH-SHA512
-sigAlgs 39      : id-hash-slh-dsa-sha2-256s-with-sha512    : SLH-DSA-SHA2-256S-WITH-SHA512
-sigAlgs 40      : id-hash-slh-dsa-sha2-256f-with-sha512    : SLH-DSA-SHA2-256F-WITH-SHA512
-sigAlgs 41      : id-hash-slh-dsa-shake-128s-with-shake128 : SLH-DSA-SHAKE-128S-WITH-SHAKE128
-sigAlgs 42      : id-hash-slh-dsa-shake-128f-with-shake128 : SLH-DSA-SHAKE-128F-WITH-SHAKE128
-sigAlgs 43      : id-hash-slh-dsa-shake-192s-with-shake256 : SLH-DSA-SHAKE-192S-WITH-SHAKE256
-sigAlgs 44      : id-hash-slh-dsa-shake-192f-with-shake256 : SLH-DSA-SHAKE-192F-WITH-SHAKE256
-sigAlgs 45      : id-hash-slh-dsa-shake-256s-with-shake256 : SLH-DSA-SHAKE-256S-WITH-SHAKE256
-sigAlgs 46      : id-hash-slh-dsa-shake-256f-with-shake256 : SLH-DSA-SHAKE-256F-WITH-SHAKE256
+sigAlgs 35      : id-hash-slh-dsa-sha2-128s-with-sha256    : SLH-DSA-SHA2-128s-WITH-SHA256
+sigAlgs 36      : id-hash-slh-dsa-sha2-128f-with-sha256    : SLH-DSA-SHA2-128f-WITH-SHA256
+sigAlgs 37      : id-hash-slh-dsa-sha2-192s-with-sha512    : SLH-DSA-SHA2-192s-WITH-SHA512
+sigAlgs 38      : id-hash-slh-dsa-sha2-192f-with-sha512    : SLH-DSA-SHA2-192f-WITH-SHA512
+sigAlgs 39      : id-hash-slh-dsa-sha2-256s-with-sha512    : SLH-DSA-SHA2-256s-WITH-SHA512
+sigAlgs 40      : id-hash-slh-dsa-sha2-256f-with-sha512    : SLH-DSA-SHA2-256f-WITH-SHA512
+sigAlgs 41      : id-hash-slh-dsa-shake-128s-with-shake128 : SLH-DSA-SHAKE-128s-WITH-SHAKE128
+sigAlgs 42      : id-hash-slh-dsa-shake-128f-with-shake128 : SLH-DSA-SHAKE-128f-WITH-SHAKE128
+sigAlgs 43      : id-hash-slh-dsa-shake-192s-with-shake256 : SLH-DSA-SHAKE-192s-WITH-SHAKE256
+sigAlgs 44      : id-hash-slh-dsa-shake-192f-with-shake256 : SLH-DSA-SHAKE-192f-WITH-SHAKE256
+sigAlgs 45      : id-hash-slh-dsa-shake-256s-with-shake256 : SLH-DSA-SHAKE-256s-WITH-SHAKE256
+sigAlgs 46      : id-hash-slh-dsa-shake-256f-with-shake256 : SLH-DSA-SHAKE-256f-WITH-SHAKE256
```

Comment:
Hmmm, where do the second names in these lines come from?  I recognise the "SLH-DSA-{SHA2,SHAKE}-nnn[fs]` names from FIPS 205 sect 11, but the rest?

Reply:
They're from the NIST "ASN.1" module.

Reply:
But that does leave the question of the *algorithm* name for these less clear. Since all I have officially is the `id-...` OID names, with no primary source for the algorithm names...

Reply:
> They're from the NIST "ASN.1" module.

That's the `id-...` names.  That wasn't what I was asking for.

For algorithm names, we've ended up making our own, see the starting comment in providers/implementations/include/prov/names.h, and then *also* adding on the ASN.1 names, as well as OIDs, when those were available.

Reply:
> > They're from the NIST "ASN.1" module.
> 
> That's the `id-...` names. That wasn't what I was asking for.
> 
> For algorithm names, we've ended up making our own, see the starting comment in providers/implementations/include/prov/names.h, and then _also_ adding on the ASN.1 names, as well as OIDs, when those were available.

If you'd like to propose a change to `objects.txt` a new PR would  be appropriate at this point, as this one is merged and closed. If you're asking a lateral question, I don't have an answer at this time.